ClientApp/viking-vault/src/UserDashboard/TransferFunds/TransferFunds.css

```diff
@@ -5,11 +5,11 @@
 }
 .transfer-funds-container{
     position: relative;
-    min-width: 25em;
-    width:60%;
+    min-width: 5em;
+    width:70%;
     height: 70%;
     top: 1rem;
-    left: 5rem;
+    left: 15%;
```

Comment:
If you wanna center use `margin: auto`

Reply:
Sounds good, doesn't work because we want it to be a little lower than the top bar